CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 2.31.0
+
+### Added
+
+* add optional strength and significance terms to elifeAssessment
+* add optional elifeAssessment to reviewed preprints snippet
+
+### Changed
+
+* introduce elifeAssessment to article-vor snippet
```

Comment:
`introduce optional`?

Reply:
Changed the wording to: move optional elifeAssessment to article-vor snippet